app/controllers/import_users_controller.rb

```diff
@@ -70,7 +77,7 @@ def valid_input_file?(imported_users, file) # rubocop:disable all
     unless imported_users.first.keys == accepted_keys
       flash[:error] = 'Unable to import CSV file. Please ensure that the '\
         'first line of the file exactly matches the sample input (username, '\
```

Comment:
the example is correct; i was going to ask about this. any spaces in the first line causes the import to fail.